edfi_api_client/edfi_client.py

```diff
@@ -51,29 +40,34 @@ def __init__(self,
         api_mode     : Optional[str] = None,
         api_year     : Optional[int] = None,
         instance_code: Optional[str] = None,
-        use_snapshot : bool = False,
 
         verify_ssl   : bool = True,
         verbose      : bool = False,
     ):
-        self.verify_ssl = verify_ssl
-        self.verbose = verbose
+        # Update logger first
+        if verbose:
+            logging.getLogger().setLevel(logging.INFO)
 
         self.base_url = base_url
         self.client_key = client_key
         self.client_secret = client_secret
+        self.verify_ssl = verify_ssl
         self.access_token: Optional[str] = None
 
         self.api_version = int(api_version)
         self.api_mode = api_mode or self.get_api_mode()
         self.api_year = api_year
         self.instance_code = instance_code
-        self.use_snapshot = use_snapshot
 
         # Build endpoint URL pieces
-        self.version_url_string = self._get_version_url_string()
+        self.version_url_string = "data/v3"
         self.instance_locator = self.get_instance_locator()
 
+        if self.api_version == 2:
+            raise NotImplementedError(
+                "Ed-Fi 2 functionality has been deprecated. Use `pip install edfi_api_client==0.2.4` for Ed-Fi 2 ODSes."
```

Comment:
Can we change this message so suggest running less than 3.0, instead of a pinned 2.x version?